src/components/text.tsx

```diff
@@ -4,6 +4,6 @@ interface IProps {
   text: string;
 }
 
-export const Text: React.FC<IProps> = ({text}) => (
-  <div>{text}</div>
+export const Text: React.FC<IProps> = ({ text }) => (
+  <div>{ text }</div>
```

Comment:
I believe adding spaces here is inconsistent with general CC practice. Changing the configuration from
```
"react/jsx-curly-spacing": ["error", { "when": "never", "children": { "when": "always" } }]
```
to
```
"react/jsx-curly-spacing": ["error", { "when": "never", "children": true }]
```
should allow what seems to me like the more widespread usage.